gulpfile.js

```diff
@@ -800,6 +800,18 @@ function getRebuildBranchName() {
   return 'rebuild_' + mm + '_' + dd + '_' + yyyy;
 };
 
+// Helper function: get a name for a rebuild branch. Format: rebuild_month_yyyy.
+function getRCBranchName() {
+  var date = new Date();
+  var monthNames = [
+    'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun',
+    'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'
+  ];
+  var month = monthNames[date.getMonth()];
+  var yy = date.getFullYear().slice(-2);
+  return 'rc_' + month + '_' + yyyy;
```

Comment:
This is different from our current naming scheme so are we going to have to update the script that creates the release doc?